index.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Reducer } from 'redux';
+
+export default function reduceReducer<S>(
+  initialState: S,
+  ...reducers: Array<Reducer<S>>
+): Reducer<S>;
```

Comment:
Do we need `Reducer` from `redux` as a dependency here? `reduce-reducers` is actually not Redux specific, so we might want to have our own `Reducer` type like `type Reducer<S> = (state: S | undefined, payload: any) => S` if I'm not wrong.

Reply:
I fixed that